packages/sdk/src/types/Validations.ts

```diff
@@ -53,6 +53,8 @@ export interface ValidationsValidateVoucherResponse {
 		discount_amount: number
 		total_discount_amount: number
 		total_amount: number
+		applied_discount_amount: number
```

Comment:
Those fields are not always returned from API.
You can have successful response from API but those fields are not going to be send back if for example none of the item is applicable to.

The same actually applies to `discount_amount` and `total_discount_amount` - it was a mistake to make them always present and fixing this would require bumping major version - I would like to avoid it for now and do it later on together with larger refactor of types.

I would gladly merge this and release new version if you could make `applied_discount_amount` and `total_applied_discount_amount` optional by adding `?` modifier to them

Reply:
sure, let me know when we're ready for a major bump because I think we also need to mark as mandatory the order.amount field (at the moment is optional, but we get an error from the API if we don't pass it in the body)

Reply:
This is because amount is required if you have validation rules based on order.
If you create simple voucher with no val rules attached you can validate without order at all.

Reply:
Ah ok got it thanks!